Bootcamp2015-AmazingRace/Helpers/MobileServiceHelper.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Bootcamp2015.AmazingRace.Base;
+using Bootcamp2015.AmazingRace.Models;
+using Microsoft.WindowsAzure.MobileServices;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Activation;
+
+namespace Bootcamp2015.AmazingRace.Helpers
+{
+    public static class MobileServiceHelper
```

Comment:
Ideally this would be implemented as a service and injected rather than using a static class. Also all methods using async should be returning a Task.